### PR TITLE
Fix group icon on desktop

### LINKED ISSUE #6747: After creating a group chat, chat icons are not displayed
## Description
*Type*: Bug
After creating a group chat, the chat icon is not shown in chat list (left panel) and chat header (top).

## Expected behavior
An icon with the first letter of the group name is shown both in chat list and chat header:

captura de pantalla 2018-11-15 a la s 4 42 48 p m

[Designs on Figma](https://www.figma.com/file/MRYBI5axcTwL4Ft8xAYhFOfy/Desktop-Chat?node-id=230%3A16017)
[Icon design specs](https://www.figma.com/file/MRYBI5axcTwL4Ft8xAYhFOfy/Desktop-Chat?node-id=309%3A11923)

## Actual behavior
Chat list:
captura de pantalla 2018-11-15 a la s 4 37 34 p m

Chat header:
captura de pantalla 2018-11-15 a la s 4 46 05 p m

### Reproduction
- Open Status Desktop
- Create a group chat

### Solution
- [ ] Add group icon to chat list
- [ ] Add group icon to chat header

### Additional Information
Wanna improve the group chat experience? Go to this 👉 [issue](https://github.com/status-im/status-react/issues/6607) 👈

### PR DESCRIPTION
**Pull request by vitvly**
Fixes #6747

### CODE REVIEW

File: src/status_im/ui/screens/desktop/main/chat/views.cljs [hunk 1]
```diff
@@ -32,10 +32,10 @@
     [react/view {:style styles/toolbar-chat-view}
      [react/view {:style {:flex-direction :row
                           :flex 1}}
-      (if public?
+      (if (or public? group-chat)
```
Comment (yenda) [inline src/status_im/ui/screens/desktop/main/chat/views.cljs:35]: if a chat is `public?` then it is `group-chat` as well

**Review by yenda — COMMENTED** [review]
I don't mind the fix but it will all go away with the refactoring could you review https://github.com/status-im/status-react/pull/6787 and see if `enrich-active-chats` fixes that ? (not reflected in views yet)

**Review by goranjovic — APPROVED** [review]

**Comment by vitvly** [comment]
@yenda thanks for the pointer! Built `refactor/chat-and-contact-subs` locally, chat list icon is fixed! But still missing in chat header. Also noticed there is an issue with unviewed messages indicator on tab icons there - it shows a number when there are zero unviewed.

File: src/status_im/ui/screens/desktop/main/chat/views.cljs (hunk 1 above)
Comment (vitvly) [reply src/status_im/ui/screens/desktop/main/chat/views.cljs:35]: Yes, but not vice versa. So needed to add additional condition here.

**Comment by vitvly** [comment]
@yenda I think that with #6787 this problem will be gone. So maybe it makes sense to close this PR.

**Review by vkjr — APPROVED** [review]

**Review by rcullito — APPROVED** [review]

**Pull request closed without merging**